data-sources/xmatters/config.yml

```diff
@@ -0,0 +1,9 @@
+id: xmatters
+displayName: xMatters
```

Comment:
Another notification channel data source, should point to workloads? Should there be a single workloads data source?